compliance/tags/azure_rg_tags/azure_resource_group_tags.pt

```diff
@@ -153,7 +164,7 @@ end
 
 policy "policy_azure_resource" do
   validate_each $ds_filtered_resources do
-    summary_template "{{ len data }} Azure Resources out of compliance. Azure Subscription ID: {{parameters.param_azure_sub}}"
+    summary_template "{{ len data }} Azure Resources out of compliance."
     detail_template <<-EOS
 # Azure Resources Incorrectly Tagged
 | Resource Name | Resource Group | Azure Region | Resource ID | Tag Value | 
```

Comment:
Please add subscription name to the result and sort by subscription name and resource name

Reply:
@cdwilhelm I have added the subscription name and sorted as well.